onc/Onc.m

```diff
@@ -1,7 +1,7 @@
 classdef Onc < onc.OncDiscovery & onc.OncDelivery & onc.OncRealTime & onc.OncArchive
-    %% ONC Facilitates access to Ocean Networks Canada's data through the Oceans 2.0 API.
-    % For detailed information and usage examples, visit our official documentation
-    % at https://wiki.oceannetworks.ca/display/CLmatlab
+    %% ONC Facilitates access to Ocean Networks Canada's data through the Oceans 3.0 API.
+    % For detailed information and usage examples, run doc command or visit MATLAB's help browser
```

Comment:
Is it possible to run "doc ONC" or something along those lines to directly open the library documentation? Instead of navigating to the documentation after running "doc"?

Reply:
yes i updated that! Now can run `doc ONC Toolbox` to view the documentation or run `help onc` to view short description of library and link to documentation.